### DISCUSSION TITLE
Enemy free "SafeZone"?

### DISCUSSION THREAD

**Discussion by CALADARI** [Vdauphin/HeartsAndMinds#1330] [Question and answer] [answered]
As title described. I migrated the game into Chernarus 2020, I took upper left airstip as base. But that is an enemy spawnpoint so have to fight our way out every time.
Is there a way I can set up a safezone to take a map location as mother base?
o7

**Comment by ghost** [comment, marked as answer]
You can place a marker on the map where you want your base, give it a variable name (I use "player_base".

Then go into the mission folder, core > fnc > city > init.sqf.

On line 66 you should see this;
 
 ​        ​/* 
 ​        //if you want a safe area 
 ​        if ((getMarkerPos "YOUR_MARKER_AREA") inArea [_position, 500, 500, 0, false]) exitWith {}; 
 ​        ​*/

Change to:
 
 ​        
 ​        //if you want a safe area 
 ​        if ((getMarkerPos "player_base") inArea [_position, 500, 500, 0, false]) exitWith {}; 

The 500's represent the distance, so you can change those to adjust the size of no spawn zone. 

Hope this helps, replying from my phone so idk how the formatting will look lol
 ​        ​

**Reply by Vdauphin** [reply]
http://vdauphin.github.io/HeartsAndMinds/Prevent-AI-from-spawning-near-the-base
